dotCMS/src/main/java/com/dotmarketing/viewtools/LanguageWebAPI.java

```diff
@@ -168,20 +166,33 @@ public String get(String key, List args) {
 	public String get(String key, String languageId) {
 		String value = null;
 		try {
-			Language lang = LanguageUtil.getUserLanguage(langAPI.getLanguage(languageId), request.getLocale());
-			value = langAPI.getStringKey(lang, key);
+
+			//Find the requested language
+			Language requestedLanguage;
+			try {
+				requestedLanguage = langAPI.getLanguage(languageId);
+			} catch (Exception e) {
+				Logger.error(this, String.format("Requested language [%s] not found.", languageId),
+						e);
+				requestedLanguage = null;
+			}
+
+			Language lang;
+			if (null != requestedLanguage) {
+				lang = LanguageUtil.getUserLanguage(requestedLanguage, request.getLocale());
+				value = langAPI.getStringKey(lang, key);
+			}
 
 			if ((!UtilMethods.isSet(value) || value.equals(key))

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "value" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2259)